### DISCUSSION TITLE
previewTextPlaceholder预览态占位符为空时，select组件多出一条线

### DISCUSSION THREAD

**Discussion by li9269391** [alibaba/formily#1628] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/10287877/122906010-4a434b00-d384-11eb-932f-dc8b4a30dfaa.png)
可否去除 select 为空时的 tag 线条

**Comment by janryWang** [comment, marked as answer]
不打算支持为空的情况了，为空不是一个优秀的交互体验
